mllib/src/main/scala/org/apache/spark/ml/r/KMeansWrapper.scala

```diff
@@ -17,14 +17,21 @@
 
 package org.apache.spark.ml.r
 
+import org.apache.hadoop.fs.Path
+import org.json4s._
+import org.json4s.DefaultFormats
```

Comment:
It looks `import org.json4s.DefaultFormats` is not needed because `import org.json4s._` imports that.